### ISSUE TITLE
Softmax Regressions: Please don't talk about probabilities

### ISSUE THREAD

**Issue by StefanoD** [tensorflow/tensorflow#9022]
[This](https://www.tensorflow.org/get_started/mnist/beginners) tutorial talks about probabilities on softmax regression. This is a common mistake. A classifier talks about its confidence, but it is not a real probability of some kind of _distribution_. Please don't teach something wrong, only to make it some kind of simple.
_Confidence_ is simple enough.

**Comment by asimshankar** [comment]
Pardon the naiveté, but isn't the [output of a softmax function used to represent a probability distribution](https://en.wikipedia.org/wiki/Softmax_function)?

**Comment by StefanoD** [comment]
Classifier can be fooled, that is why you can't see them as real probabilities.
In fact, something like this a doctorand told me. But to be honest, I am lacking sources right now.

**Comment by asimshankar** [comment]
I'm going to go ahead and close this issue out for now. If you strongly disagree, please feel free to submit a PR to fix this - and describe the justification in some detail. Thanks much!

(The file to change would be: [`tensorflow/tensorflow/docs_src/get_started/mnist/beginners.md`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/get_started/mnist/beginners.md))
